lib/src/dynamics/fixture_def.dart

```diff
@@ -3,12 +3,15 @@ import '../../forge2d.dart';
 /// A fixture definition is used to create a fixture. This class defines an abstract fixture
 /// definition. You can reuse fixture definitions safely.
 class FixtureDef {
+  FixtureDef({required this.shape, this.density = 0.0});
```

Comment:
I think density should be left out of here, or all of the other parameters added too.

Reply:
Yes, I removed density.